mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -4651,6 +4683,37 @@ resource "google_container_cluster" "with_net_ref_by_name" {
 `, network, cluster, cluster)
 }
 
+func testAccContainerCluster_autoprovisioningDefaultsManagement(clusterName string, autoUpgrade, autoRepair bool) string {
+	return fmt.Sprintf(`
+resource "google_container_cluster" "with_autoprovisioning_management" {
+  name               = "%s"
+  location           = "us-central1-f"
+  initial_node_count = 1
+
+  cluster_autoscaling {
+    enabled = true
+
+	resource_limits {
+	  resource_type = "cpu"
+      maximum       = 2
```

Comment:
nit: would you align this line as well please?

Reply:
Fixed